operators/vision/image_encoder_darwin.hpp

```diff
@@ -95,6 +95,7 @@ struct EncodeImage {
     CFDataGetBytes(result, CFRangeMake(0, size), *outbuffer);
     CFRelease(result);
     *outsize = size;
+    return {};
```

Comment:
Missing the return results in `EXC_BREAKPOINT (code=1, subcode=0x10000e698)` on ARM64.